app/policies/hour_policy.rb

```diff
@@ -0,0 +1,15 @@
+class HourPolicy < ApplicationPolicy
+  class Scope < ApplicationScope
+    def resolve
+      all if superadmin?
```

Comment:
The volunteer hours are handled with volunteer_hours action and views. That's why the index is available only to the superadmin.